src/main/resources/config/liquibase/changelog/20190707025824_changelog.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+    <changeSet author="phuong.anh.nguyen (generated)" id="1562461116856-23">
+        <addColumn tableName="jhi_user">
+            <column name="guided_tour_settings" type="varchar(255)"/>
```

Comment:
I wonder if 255 characters might be not enough at some time... Could be a good choice for longtext.
You might want to check how big it is atm.

Reply:
@thilo-behnke Yes that's a good point. I changed that to text instead of longtext because that should be more than enough characters.